backend/src/data/repositories/podcast/podcast.repository.ts

```diff
@@ -1,10 +1,17 @@
-import { Podcast as TPodcast, PodcastCreatePayload } from '~/common/types/types';
+import { Podcast as TPodcast } from '~/common/types/types';
 import { PodcastModel as PodcastM } from '~/data/models/models';
 
 type Constructor = {
   PodcastModel: typeof PodcastM;
 };
 
+type PodcastRecordPayload = {
```

Comment:
```suggestion
type PodcastCreateDTOPayload = {
```
move it to the common folder on the backend